ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  ghostFS:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      PerformanceMode: generalPurpose
+      Encrypted: false
+  ghostMount0:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 0 
+        - !Ref DefaultVPCSubnets
+  ghostMount1:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 1 
+        - !Ref DefaultVPCSubnets
+  ghostMount2:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 2 
+        - !Ref DefaultVPCSubnets
+  ghostMount3:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 3 
+        - !Ref DefaultVPCSubnets
+  ghostAPI:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      ContainerDefinitions:
+        - Image: registry.hub.docker.com/library/ghost
+          PortMappings:
+            - ContainerPort: 2368
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref ghostAPILogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: logs
+          MountPoints:
+            - 
+              ContainerPath: /var/lib/ghost/content
+              SourceVolume: ghostVolume
+          Name: 0
+          Environment:
+            - Name: url
+              Value: !Join [ '', [ 'http://', !GetAtt Loadbalancer.DNSName ] ]
```

Comment:
Out of curiousity, does it need to be able to hit itself? Why is this provided to the container?

Reply:
Certain Ghost functionality depends on `url` which must be the public URL for the CMS. For example, image serving and in-console site previews. https://ghost.org/docs/concepts/config/